nsis/src/nsis/file/NsisConstants.java

```diff
@@ -17,4 +17,11 @@
 	public enum BlockHeaderType {
 		PAGES, SECTIONS, ENTRIES, STRINGS, LANGTABLES, CONTROL_COLORS, BACKGROUND_FONT, DATA
 	}
+
+	// Ghidra memory block names
+	public static final String firstHeaderMemoryBlockName = ".first_header";
```

Comment:
I think these should be all capitals so they stand out as constants:

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

(Don't forget to use "refactor" in eclipse for easy renaming :-)

Reply:
oups yes they should